src/main/java/uk/gov/hmcts/darts/event/service/handler/SentencingRemarksAndRetentionPolicyHandler.java

```diff
@@ -31,17 +32,18 @@ public SentencingRemarksAndRetentionPolicyHandler(RetrieveCoreObjectService retr
                                                       HearingRepository hearingRepository,
                                                       CaseRepository caseRepository,
                                                       ApplicationEventPublisher eventPublisher,
-                                                      TranscriptionsApi transcriptionsApi) {
-        super(retrieveCoreObjectService, eventRepository, hearingRepository, caseRepository, eventPublisher);
+                                                      TranscriptionsApi transcriptionsApi,
+                                                      AuthorisationApi authorisationApi) {
+        super(retrieveCoreObjectService, eventRepository, hearingRepository, caseRepository, eventPublisher, authorisationApi);
         this.transcriptionsApi = transcriptionsApi;
     }
 
     @Override
     @Transactional
     public void handle(final DartsEvent dartsEvent, EventHandlerEntity eventHandler) {
         var transcriptionRequestDetails = transcriptionRequestDetailsFrom(
-            dartsEvent,
```

Comment:
Do we want to add info/debug level logging across

Reply:
i wasn't actually changing this handler, i just changed the constructor, that affected all handlers.